doc/apiv3-documentation.apib

```diff
@@ -1474,6 +1478,94 @@ This is an example of how a form might look like. Note that this endpoint does n
                 "message": "The resource you are about to edit was changed in the meantime."
             }
 
+# Group MyPreferences
+
+## Linked Properties
+|  Link     | Description                                              | Type          | Constraints           | Supported operations |
+|:---------:|----------------------------------------------------------| ------------- | --------------------- | -------------------- |
+| self      | This MyPreferences                                       | MyPreferences | not null              | READ                 |
+| user      | The current user that this preference belongs to         | User          | not null              | READ                 |
+
+## Local Properties
+| Property               | Description                                                | Type       | Constraints | Supported operations |
+|:----------------------:| -----------------------------------------------------------| ---------- | ----------- | -------------------- |
+| hideMail               | Hide mail address from other users                         | Boolean    |             | READ / WRITE         |
+| timeZone               | Current selected time zone                                 | String     |             | READ / WRITE         |
+| theme                  | Current selected theme                                     | String     |             | READ / WRITE         |
+| commentSortDescending  | Sort comments in descending order                          | Boolean    |             | READ / WRITE         |
+| warnOnLeavingUnsaved   | Issue warning when leaving a page with unsaved text        | Boolean    |             | READ / WRITE         |
+| accessibilityMode      | Enable accessibility mode                                  | Boolean    |             | READ / WRITE         |
+
+## MyPreferences [/api/v3/my_preferences]
+
++ Model
+    + Body
+
+            {
+                "_type" : "MyPreferences",
+                "_links" : {
+                    "self" : {
+                        "href" : "/api/v3/my_preferences",
+                    },
+                    "user": {
+                        "href": "/api/v3/users/1",
+                        "title": "John Sheppard"
+                    }
+                },
+                "hideMail" : false,
+                "timeZone" : "Berlin",
```

Comment:
I've simply taken that from the available options under `/my`, but its true that we need to think about validating these values.